frontend/viewer/src/ShadcnProjectView.svelte

```diff
@@ -34,6 +35,9 @@
   initView();
   initViewSettings();
 
+  // Load idle service into component context for inputs to use
+  const _idleService = useIdleService();
```

Comment:
we're getting a lint error here now

Reply:
I thought the initial `_` would be enough for the `no-unused-vars` check to skip it, but apparently not. I'll have to disable the no-unused-vars check for this line, or else do `void useIdleService()`.

Reply:
Done in commit 0ebc0eb6b.